.gitignore

```diff
@@ -0,0 +1,49 @@
+# Dependencies
```

Comment:
I'd recommend removing unnecessary items from gitignore; only add what's needed for now